src/components/FormValidator.js

```diff
@@ -1,5 +1,5 @@
 class FormValidator {
-  constructor(settings, formElement) {
+  constructor(formElement, settings) {
```

Comment:
There is abn error in the console when I open the page `FormValidator.js:9 Uncaught TypeError: Cannot read properties of null (reading 'querySelector')
    at new FormValidator (FormValidator.js:9:38)
    at index.js:129:26
    at main.js:8039:2
    at main.js:8040:12`

Maybe because of that I don't see the cards